scripts/run_integration_tests.sh

```diff
@@ -65,49 +64,48 @@ function remove_stack() {
 }
 trap remove_stack EXIT
 
-echo "Deploying functions"
+echo "Deploying functions for runtime $SERVERLESS_RUNTIME"
```

Comment:
Nit: I would add a check with a helpful error message in case we forget to specify `$SERVERLESS_RUNTIME`